bin/rspec

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env jruby
+#!/usr/bin/env -S jruby -J-Djdk.util.zip.disableZip64ExtraFieldValidation=true
```

Comment:
We should be setting this property in karaf.rb so that applies when you run console as well

Reply:
I tried that. It didn't work.

Reply:
This property needs to be set very early on, i.e. when loading java itself. Setting it inside karaf.rb is too late.

This comment might explain why:
https://github.com/iBotPeaches/Apktool/issues/3174#issuecomment-1653200079